src/mmw/js/src/core/utils.js

```diff
@@ -433,6 +433,13 @@ var utils = {
             });
             return newRow;
         });
+    },
+
+    calculateVisibleRows: function(minScreenHeight, avgRowHeight, minRows) {
+        var screenHeight = document.documentElement.clientHeight;
+        return Math.max(
+            Math.floor((screenHeight - minScreenHeight) / avgRowHeight) || minRows,
```

Comment:
Out of curiosity, what's the condition in which `|| minRows` would get triggered here?